CMakeLists.txt

```diff
@@ -64,7 +64,7 @@ list(GET LLVM_VERSION_LIST 1 LLVM_MINOR_VERSION)
 set(LLVM_MAJOR_VERSION "${LLVM_MAJOR_VERSION}")
 set(LLVM_MINOR_VERSION "${LLVM_MINOR_VERSION}")
 
-set(REMILL_LLVM_VERSION "${LLVM_MAJOR_VERSION}")
+set(REMILL_LLVM_VERSION "${LLVM_MAJOR_VERSION}.${LLVM_MINOR_VERSION}")
```

Comment:
Don't forget remillConfig.cmake.in ;-)

Reply:
I changed the other file. Please let me know if that is correct (if the CI is not checking it). Sadly I don't have a quick local setup to test this.